sunpy/map/header_helper.py

```diff
@@ -119,6 +131,15 @@ def make_fitswcs_header(data, coordinate, reference_pixel: u.pix = None,
     meta_wcs['cdelt1'], meta_wcs['cdelt2'] = (scale[0]*meta_wcs['cunit1']/u.pixel,
                                               scale[1]*meta_wcs['cunit2']/u.pixel)
 
+    if rotation_angle is not None:
+        meta_wcs['crota2'] = rotation_angle.to_value()
```

Comment:
We should convert this to a `PCi_j` matrix and store it in the header that way. `CROTA` is deprecated in the FITS standard.

Reply:
but I guess it's done in mapbase - so would it not just be duplicating the code?

Reply:
yeah it would be, but I think this function should generate the most compliant FITS header it can.